src/Proxy/SpyProxy.php

```
@@ -23,19 +24,32 @@
-    protected function pureMethod(\ReflectionMethod $method): string
+    protected function pureMethod(ReflectionMethod $method, bool $isHook = false): string
     {
-        return $this->impureCall($method);
+        return $this->impureCall($method, $isHook);
     }
 
-    protected function impureCall(\ReflectionMethod $method): string
+    protected function impureCall(ReflectionMethod $method, bool $isHook = false): string
     {
-        $name = $method->getName();
+        $getHook = true;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$getHook'. (Unused Code Rules)

(UnusedLocalVariable)